### ISSUE TITLE
Mac v4.0: Keyboard shortcuts don't work and don't display in menus

### ISSUE THREAD

**Issue by jamesritzman** [transmission/transmission#4920]
### What is the issue?

Keyboard shortcuts do not work.
In the app's dropdown menus, the keyboard shortcut reminders/hints do not appear, instead showing as boxes with question marks.
Mac OS 12.6.2 on a Mac Studio (2022 - M1 Max)
![Screen Shot 2023-02-16 at 12 42 46 PM](https://user-images.githubusercontent.com/5994624/219483257-3c688ecf-19ac-49cb-89ef-80afa757bc5a.png)

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0.0 (280ace12f8)

**Comment by Coeur** [comment]
Duplicate of #4041.
Most people (including me) have never ever encountered this issue.

Just clean your build folder, rebuild and reinstall the app.

**Comment by ckerr** [comment]
Closing as a duplicate of #4041 as per previous comment

**Comment by jamesritzman** [comment]
Thanks for the info re: cleaning the build folder, but I never built the app locally.  I obtained it via an installer, iirc.
After quitting and relaunching the app, the menus were back to normal as was keyboard shortcut functionality.
This was all good for a few days, but now the app is back to a non-working state in regards to the keyboard shortcuts, the menus are again showing boxes with question marks, and selecting items from the menus does not work either.
I can post this information on the other ticket (4041) as well.

**Comment by Coeur** [comment]
@ckerr I'll re-open this issue as the old one was closed since the originator was not reproducing it anymore.

**Comment by whaler-ragweed** [comment]
I'm using Transmission CLI 4.0.1 (installed from Homebrew) with web interface. The keyboard shortcuts do not work at all.

**Comment by ckerr** [comment]
@whaler-ragweed the web client shortcuts is covered in https://github.com/transmission/transmission/issues/4829

**Comment by jamesritzman** [comment]
> @ckerr I'll re-open this issue as the old one was closed since the originator was not reproducing it anymore.
> 
> @jamesritzman and you've never seen this in any other application, right?

Correct.  Also, I upgraded to 4.0.2 (2a57b17031) recently, and am experiencing the issue again.  I did not notice the issue immediately after upgrading, but after launching the app today the keyboard shortcuts do not work and the menu items have "?"s.
